xml/System.Numerics/Vector.xml

```diff
@@ -4196,8 +4212,8 @@ Note that this method returns a <xref:System.Single> instead of an integral type
         <param name="source">The source vector whose elements are widened into the outputs.</param>
         <param name="dest1">The first output vector, whose elements will contain the widened elements from lower indices in the source vector.</param>
```

Comment:
```suggestion
        <param name="dest1">When this method returns, contains the widened elements from lower indices in the source vector.</param>
```

Reply:
Makes sense. "First and second output vector" sound redundant.